.gitignore

```diff
@@ -140,3 +140,7 @@ contrib/macos/dosbox-x.plist
 /dosbox-x.conf
 /src/dosbox-x.conf
 
+src/dos/dos_programs.7z
```

Comment:
Do we need to ignore these files?
There are no such file in the current code tree.

Reply:
No, you don't. In fact those files came by mistake from another code tree of mine, but they hadn't to.